src/common/middlewares/jwt.middleware.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { NextFunction, Request, Response } from 'express';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class JwtMiddleware implements NestMiddleware {
+  constructor(private readonly jwtService: JwtService) {}
+
+  async use(req: Request, res: Response, next: NextFunction) {
+    const token = req.headers['authorization'];
+
+    if (!token) {
+      return res
+        .status(401)
```

Comment:
I think it is better to use standard HTTP constants rather than `401` directly.

Reply:
For reference:
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
- https://www.npmjs.com/package/http-status-codes
- https://stackoverflow.com/questions/25146687/list-of-node-js-http-status-codes